front_end/Matching/MatchingBrowseTwo.js

```diff
@@ -1,50 +1,55 @@
 //Matching-Browse-2
 import React, {Component} from 'react';
 import {
-    SafeAreaView,
-    View,
-    Text,
-    TextInput,
-    TouchableOpacity,
+  SafeAreaView,
```

Comment:
I noticed for MatchingBrowseOne.js  and QuestionList.js you compressed these libraries into one line – can we keep this as one line as well then?

Reply:
update: I believe @shlee2017 actually mentioned it'd be more readable if each library had its own line – could you change the other 2 files instead?

Reply:
Disabled Prettier - Code formatter to do this.